src/formElementPlugins/SummaryViewWrapper.tsx

```diff
@@ -58,7 +58,11 @@ const SummaryViewWrapper: React.FC<SummaryViewWrapperProps> = ({
   if (!pluginCode || !isVisible) return null
 
   const DefaultSummaryView: React.FC = () => {
-    const combinedParams = { ...simpleParameters, ...evaluatedParameters }
+    const combinedParams = {
+      ...simpleParameters,
+      ...evaluatedParameters,
+      ...response?.evaluatedParameters,
+    }
```

Comment:
We still evaluate parameters (in `evaluatedParameters`), cos why not? It's possible we may want to have dynamically evaluated parameters in a DataView Summary element, in which case it's still possible.